.github/workflows/test.yaml

```diff
@@ -0,0 +1,48 @@
+on: [push]
+
+name: ci
+
+jobs:
+  check:
+    name: Check
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: nightly
```

Comment:
Should we pick a particular minimum rust version to support?

Reply:
I don't have a preference! Let me know what you'd like to do.

Reply:
Nightly is fine for now.